pkg/kubelet/kubelet_test.go

```diff
@@ -2350,7 +2350,7 @@ func TestSyncTerminatingPodKillPod(t *testing.T) {
 }
 
 func TestPreInitRuntimeService(t *testing.T) {
-	err := PreInitRuntimeService(nil, nil, nil, "", "", "", "", "")
+	err := PreInitRuntimeService(nil, nil, nil, "", "", "", "")
```

Comment:
this function is only called for this test?

Reply:
Also here:
https://github.com/kubernetes/kubernetes/blob/3b2824ea683d6bceba8fba932a8de5f86f72c8af/cmd/kubelet/app/server.go#L799-L804

Reply:
`dockerless` build tag has been removed in https://github.com/kubernetes/kubernetes/pull/107119.